package.json

```diff
@@ -18,13 +18,15 @@
     "@backstage-community/cli": "portal:./workspaces/repo-tools/packages/cli",
     "@backstage/cli": "^0.27.0",
     "@backstage/cli-node": "^0.2.2",
+    "@backstage/errors": "^1.2.5",
     "@changesets/parse": "^0.4.0",
     "@manypkg/get-packages": "^2.2.2",
     "@octokit/rest": "^20.1.1",
     "@spotify/eslint-plugin": "^15.0.0",
     "@spotify/prettier-config": "^15.0.0",
     "array-to-table": "^1.0.1",
     "codeowners-utils": "^1.0.2",
+    "cross-fetch": "^4.0.0",
```

Comment:
don't think these packages should be in the root package.json, should be in the workspace deps instead.

Reply:
Yes you are right! I don't know why are there probably vscode suggestion I will remove it